tests/lib/snaps/test-snapd-framebuffer/meta/snap.yaml

```diff
@@ -0,0 +1,12 @@
+name: test-snapd-framebuffer
+version: 1.0
+summary: Basic snap to check access to file system
+description: A basic snap to check access to file system
```

Comment:
Both summary and description need changing, looks like a copy-paste from another test.

Reply:
done